functional-test/src/test/groovy/org/rundeck/tests/functional/selenium/jobs/JobActivityHistorySpec.groovy

```diff
@@ -0,0 +1,82 @@
+package org.rundeck.tests.functional.selenium.jobs
+
+import org.openqa.selenium.By
+import org.openqa.selenium.WebElement
+import org.rundeck.util.annotations.SeleniumCoreTest
+import org.rundeck.util.container.SeleniumBase
+import org.rundeck.util.container.SeleniumContext
+import org.rundeck.util.gui.pages.login.LoginPage
+import org.rundeck.util.gui.pages.home.HomePage
+import org.rundeck.util.gui.pages.jobs.JobCreatePage
+import org.rundeck.util.gui.pages.execution.ExecutionShowPage
+import org.rundeck.util.gui.pages.execution.HtmlRenderedOutputPage
+import org.rundeck.util.gui.pages.jobs.JobListPage
+import org.rundeck.util.gui.pages.activity.ActivityPage
+import org.rundeck.util.gui.pages.jobs.JobTab
+import org.rundeck.util.gui.pages.jobs.JobShowPage
+import spock.lang.Stepwise
+@SeleniumCoreTest
+@Stepwise
+class JobActivityHistorySpec extends SeleniumBase {
+    def setupSpec() {
+        setupProject(SELENIUM_BASIC_PROJECT)
+    }
+    def setup() {
+        def loginPage = go LoginPage
+        loginPage.login(TEST_USER, TEST_PASS)
+        println "Login successful"
+    }
+    def "create job has basic fields"() {
```

Comment:
this test is not related to your ticket AC

Reply:
> this test is not related to your ticket AC

Thank you for pointing that out. I initially thought the test needed to include the flow of creating a job and then executing it. I have removed it